tests/datastore/test_targets.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from mlrun.datastore import StreamTarget
+from mlrun.feature_store import FeatureSet
+
+
+class MockGraph:
+    def __init__(self):
+        self.args = None
+        self.kwargs = None
+
+    def add_step(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+
+def test_stream_target():
```

Comment:
Please rename the test (or/and add comment) to explain that what is actually being tested (that existence of run_id attribute does not effect the stream_path).